sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1,5 @@
+select "filmId",
+        "title"
+  from "films"
+  order by "rentalRate" asc
+  limit 50;
```

Comment:
This does not guarantee that these films will cost less than a dollar:

---

![image](https://user-images.githubusercontent.com/16143771/130702436-11dcf469-b548-4ef4-9f86-e583ffe45b43.png)